### PR TITLE
Team Himanshu

### PR DESCRIPTION
**Pull request by Himanshuch8055**
# Hackathon Submission: DevBoard

## Name of Team Lead
_Enter your full name_

Himanshu Chauhan

## GitHub handles of Team Members  
_Enter the GitHub handles of all your team members (including yourself) as separate bullet points_

- @rahulsharma123
- @priyasingh456
- @amitpatel789
- @nehasrivastava234

### Are all of your team members registered on our [Hackathon Signup page](https://apwr.dev/htf24-hackathon)?
_This is a mandatory step for all participants to be considered eligible for judging_

- [x] Yes
- [ ] No

---

## Rules and Code of Conduct  
By submitting this PR, you agree to follow our Rules and Code of Conduct.

- [x] Yes

---

## Anything Else?  
_Any other feedback, queries or information you would like to share with us?_

We're thrilled to participate in this hackathon and showcase our project, DevBoard. We've put a lot of effort into creating a solution that we believe can make a real difference in project management for development teams. We're looking forward to your feedback and the opportunity to learn from this experience. Thank you for organizing this event!

### CODE REVIEW

File: submissions/DevBoard.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+We utilized Appwrite Storage for file management within projects. This allowed team members to easily upload, store, and share project-related documents and assets.
+
+- Appwrite Functions
+
+Appwrite Functions were used to implement custom backend logic, such as automated notifications and task assignments, enhancing the overall functionality of DevBoard.
+
+## Project Repo  
+_Share a public repo link of your project_
+
+https://github.com/Discord-Group-Project/DevBoard
```
Comment (adityaoberai) [inline submissions/DevBoard.md:61]: Please make the GitHub repo of the project public @Himanshuch8055

**Review by adityaoberai — CHANGES_REQUESTED** [review]

File: submissions/DevBoard.md (hunk 1 above)
Comment (Himanshuch8055) [reply submissions/DevBoard.md:61]: > Please make the GitHub repo of the project public @Himanshuch8055 

Sorry, I will make it public. Give me some time!

Comment (adityaoberai) [reply submissions/DevBoard.md:61]: No problem! Make sure to make it public before Oct 31st and re-request a PR review  from me once it is ready

**Comment by adityaoberai** [comment]
The hackathon project repo was not made accessible by the time the hackathon concluded, which renders this submission ineligible.

image

**Pull request closed without merging**